TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0"
+          "version": "1.0.0-servicing-004500"
```

Comment:
> If you are worried about testing of the just-built Microsoft.NETCore.App as well -- it is broken for standalone apps as NuGet resolves it from the feed with 1.0.0 and not 1.0.0-servicing-*.

I am worried about testing this for the standalone case. Unless the servicing fix goes all the way upstack (is that the plan?), this will be the only coverage. 

Does leaving it at `1.0.0-servicing-004500` resolve to the proper package for standalone?